kernel/rtlil.cc

```diff
@@ -214,6 +217,12 @@ RTLIL::Const::Const(const std::string &str)
 
 RTLIL::Const::Const(int val, int width)
 {
+	if (width < 0)
```

Comment:
This should probably be a `log_assert` rather than `log_error` since this case isn't supposed to ever happen.